client/my-sites/stats/activity-log/provisioning-notice.jsx

```diff
@@ -0,0 +1,25 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'gridicons';
+
+export const ProvisioningNotice = ( { translate } ) => (
+	<CompactCard highlight="info" className="activity-log__provisioning-notice">
+		<span className="activity-log__provisioning-notice-icon">
+			<Gridicon icon="history" size={ 24 } />
+		</span>
+		<span className="activity-log__provisioning-notice-text">
+			{ translate( 'We are currently configuring your site to start protecting it with Rewind.' ) }
```

Comment:
@nicoleckohler could you recommend something for here? We're trying to show that Rewind is getting setup and there's nothing the customer needs to do but it's not quite ready yet.

Reply:
@dmsnell Is there any way you could show me the context (i.e. a screenshot of where/when this would appear)?

Reply:
(Even a rough mockup or a description would be totally fine)

Reply:
@nicoleckohler someone is looking at the Activity Log, they already provided credentials to get Rewind setup (or granted permission). They are back on the Activity Log but the Rewind buttons are disabled. They are disabled because our servers are still getting their site initialized. At some point, presumably soon, their site will be configured and Rewind will be available.

Reply:
So maybe something like:

> Your site is currently being configured. There's nothing else you need to do right now -- check back soon to see your updated Activity Log or restore backups.

Does this make sense?

Reply:
thanks @nicoleckohler! super awesome :smile: